integration/Makefile

```diff
@@ -1,4 +1,5 @@
 CONTAINER_REPO ?= ghcr.io/kinvolk/inspektor-gadget
+TEST_CONTAINER_REPO ?= $(CONTAINER_REPO)-test
```

Comment:
It is missing to update https://github.com/kinvolk/inspektor-gadget/blob/3da7b2cfc76a75b28095c246c5593314dacc4815/integration/gadget-integration-tests-job.yaml#L38